.github/jobs/win32.yml

```diff
@@ -70,6 +70,12 @@ jobs:
           Playground app:///Scripts/validation_native.js 
         displayName: 'Validation Tests'
 
+      - script: |
+          cmake --build build${{ variables.solutionName }} --target INSTALL --config RelWithDebInfo
+          cd Install/Test
+          cmake . -DBINARY_DIR=../../build${{ variables.solutionName }} ${{ variables.jsEngineDefine }} && cmake --build . --config RelWithDebInfo
```

Comment:
nit: Is it necessary for these two cmake commands to be on one line?

Reply:
nop. changed to 2 lines for clarity